src/Http/RequestError.php

```diff
@@ -38,4 +39,17 @@ public function getResponse(): ?Response
     {
         return $this->response;
     }
+
+    protected function formatResponse(array $json): string
```

Comment:
Discutabel deze method maar Procura gebruikt geen 'title' en 'detail' in hun response wanneer er een fout optreedt. Ik ga dit bij hen aangeven maar betwijfel of zij dit gaan aanpassen.

Reply:
Klopt dit heb ik aangegeven met als antwoord: Hoe graag ik het ook zou willen, maar dit kunnen wij niet op korte termijn aanpassen.

Ik haal de method wel weg zodra ik kan mergen, voor nu is ie handig om te debuggen.